packages/curvenote/src/sync/clone.ts

```diff
@@ -39,26 +38,29 @@ export async function interactiveCloneQuestions(
     }
   }
   let path: string;
-  const defaultPath = join('content', project.data.name);
+  const defaultPath = '.';
```

Comment:
This seems fine - is there a worry we clobber existing stuff in the current folder though?

Reply:
we're ok for now I think